### PR TITLE
EX-29, EX-19, EX-20, EX-35 company model done 

### PR DESCRIPTION
**Pull request by tugbaalparslan**
Admin can,

create, get, delete, and update a company

### CODE REVIEW

File: models/company.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+#     education = 'education'
+
+class CompanyModel(db.Model):
+    __tablename__ = 'company'
+
+    id = db.Column(db.Integer, primary_key=True)
+    us_employer_id = db.Column(db.String(50))
+    company_name = db.Column(db.String(200))
+    # business_area = db.Column(db.Enum(BusinessChoices), nullable=False)  # ******* this should be enum *******
+    business_area = db.Column(db.String(50), nullable=False)
```
Comment (erdemalparslan) [inline models/company.py:17]: why do we have `nullable=False` for only this field @tugbaalparslan ?

File: models/company.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    number_of_employees = db.Column(db.Integer)
+    phone = db.Column(db.String(11))
```
Comment (erdemalparslan) [inline models/company.py:19]: we don't need to restrict sizes of phone fields. There is no enforcement in verification of phone number format so let's accept whatever string the clients send us

Comment (erdemalparslan) [reply models/company.py:19]: oh sorry, forget it, I saw you restricted in validation function

**Review by erdemalparslan — APPROVED** [review]
looks good! only a comment

File: models/company.py (hunk 1 above)
Comment (tugbaalparslan) [reply models/company.py:17]: I tried to see how it works by restricting it thru SqlAlchemy. I already restricted the rest of the fields through parser.

**Pull request merged**